tests/unit/test_pareto.py

```diff
@@ -18,7 +18,8 @@
 import tensorflow as tf
 
 from tests.util.misc import TF_DEBUGGING_ERROR_TYPES, SequenceN
-from trieste.utils.pareto import Pareto, non_dominated
+from trieste.acquisition.multi_objective.dominance import non_dominated
```

Comment:
these tests are in the wrong directory. Unit tests directory structure should mirror source code directory structure